drivers/sensor/adi/adxl367/adxl367_decoder.c

```diff
@@ -75,7 +75,7 @@ static inline void adxl367_temp_convert_q31(q31_t *out, const uint8_t *buff,
 		}
 		break;
 
-	case ADXL367_14B_CHID:
+	case ADXL367_14B_CHID: {
 		uint16_t *tmp_buf = (uint16_t *)buff;
```

Comment:
another option would be to declare `uint16_t *tmp_buf` outside of the switch, would avoid the braces here, up to you I guess